tests/admin-mode_test.go

```diff
@@ -0,0 +1,273 @@
+
+			//Creating ChaosEngine
+			By("Creating ChaosEngine")
+			err = exec.Command("kubectl", "apply", "-f", experimentName+"-ce.yaml", "-n", chaosTypes.ChaosNamespace).Run()
+			Expect(err).To(BeNil(), "Failed to create chaos engine")
+			fmt.Println("ChaosEngine for pod delete created successfully")
+			time.Sleep(2 * time.Second)
+
+			//Fetching the runner pod and Checking if it gets in Running state or not
+			By("Wait for runner pod to come in running sate")
+			runnerPodStatus, err := utils.RunnerPodStatus(chaosTypes.ChaosNamespace, engineName, client)
+			Expect(runnerPodStatus).NotTo(Equal("1"), "Faileded to get in running state")
+			Expect(err).To(BeNil(), "Failed to get the runner pod")
+			fmt.Println("Runner pod for pod delete is in Running state")
+
+			//Waiting for experiment job to get completed
+			//Also Printing the logs of the experiment
+			By("Waiting for job completion")
+			jobPodLogs, err := utils.JobLogs(experimentName, chaosTypes.ChaosNamespace, engineName, client)
+			Expect(jobPodLogs).To(Equal(0), "Failed to print the logs of the experiment")
+			Expect(err).To(BeNil(), "Failed to get the experiment job pod")
+
+			//Checking the chaosresult
+			By("Checking the chaosresult")
+			chaosResult, err := clientSet.ChaosResults(chaosTypes.ChaosNamespace).Get(engineName+"-"+experimentName, metav1.GetOptions{})
+			Expect(string(chaosResult.Status.ExperimentStatus.Verdict)).To(Equal("Pass"), "Verdict is not pass chaosresult")
+			Expect(err).To(BeNil(), "Failed to get chaosresult")
+		})
+	})
+	// BDD TEST CASE 2
+	// Operator in litmus ns, engine and experiment is in test ns
+	// Application is in deafult ns
+	Context("Check for litmus components", func() {
+
+		It("Should check for creation of runner pod", func() {
+
+			//Create Namespace for the test
+			By("Creating namespace for the test")
+			cmd := exec.Command("kubectl", "create", "ns", "test")
+			cmd.Stderr = os.Stderr
+			err := cmd.Run()
+			if err != nil {
+				log.Fatalf("cmd.Run() failed with %s\n", err)
+			}
+
+			//Installing RBAC for experiment that is pod-delete
+			rbacPath := "https://raw.githubusercontent.com/litmuschaos/pages/master/docs/litmus-admin-rbac.yaml"
+			err = exec.Command("wget", "-O", experimentName+"-sa.yaml", rbacPath).Run()
+			Expect(err).To(BeNil(), "Fail to fetch rbac")
+			err = exec.Command("sed", "-i",
+				`s/namespace: litmus/namespace: test/g;
+			     s/name: litmus-admin/name: admin-rbac/g`,
+				experimentName+"-sa.yaml").Run()
+			err = exec.Command("kubectl", "apply", "-f", experimentName+"-sa.yaml").Run()
+			Expect(err).To(BeNil(), "Fail to create RBAC")
+			fmt.Println("Rbac has been created successfully !!!")
+
+			//Creating Chaos-Experiment for pod delete
+			By("Creating Experiment for pod delete")
+			err = exec.Command("wget", "-O", "pod-delete.yaml", "https://hub.litmuschaos.io/api/chaos?file=charts/generic/pod-delete/experiment.yaml").Run()
```

Comment:
We already get this file in BDD1, so it is already there in local

Reply:
tmp file ... it will contain new contents